release notes/v0.49.0.md

```diff
@@ -0,0 +1,202 @@
+k6 `v0.49.0` is here 🎉! This release includes:
+
+- a built-in web-dashboard displaying test results in real-time
+- add `clear` functionality to the `locator` classes within the browser module
+- gRPC experimental module being merged back
+- being able to get the selection from an element in `k6/html`
+- collect internal modules and outputs used by a script
+- getting `k6/experimental/timers` ready to be stabilized
+- a bunch of internal changes, most of which are fixing lint errors, but also making MacOS test work :tada: 
+
+
+## Breaking changes
+
+
+- [#3494](https://github.com/grafana/k6/pull/3494) stops pushing `loadimpact/k6` docker image. If you still use it please migrate to the  `grafana/k6`
+- [browser#1111](https://github.com/grafana/xk6-browser/pull/1111) removes `timeout` option for `isVisible` and `isHidden` since the API no longer waits for the element to appear on the page.
+
+
+## New features
+
+### Web Dashboard
+
+The new web dashboard brings real-time visualization to load testing. This feature allows users to monitor test progress and analyze
```

Comment:
```suggestion
The new [web dashboard]() brings real-time visualization to load testing. This feature allows users to monitor test progress and analyze
```
Are we going to add a link to the documentation?

Reply:
Yes, I'm working on the docs at the moment, and figuring out with Heitor how to structure it. I'll update the link as soon as it's ready 👍🏻